# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../attr/destdir.patch
# Copyright (C) 2007 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- ./include/builddefs.in.orig	2007-01-25 01:32:46.000000000 -0300
+++ ./include/builddefs.in	2007-06-25 05:44:53.000000000 -0400
@@ -27,13 +27,13 @@
 PKG_VERSION	= @pkg_version@
 PKG_PLATFORM	= @pkg_platform@
 PKG_DISTRIBUTION= @pkg_distribution@
-PKG_BIN_DIR	= @bindir@
-PKG_SBIN_DIR	= @sbindir@
-PKG_LIB_DIR	= @libdir@@libdirsuffix@
-PKG_DEVLIB_DIR	= @libexecdir@@libdirsuffix@
-PKG_INC_DIR	= @includedir@/attr
-PKG_MAN_DIR	= @mandir@
-PKG_DOC_DIR	= @datadir@/doc/@pkg_name@
+PKG_BIN_DIR	= $(DESTDIR)@bindir@
+PKG_SBIN_DIR	= $(DESTDIR)@sbindir@
+PKG_LIB_DIR	= $(DESTDIR)@libdir@@libdirsuffix@
+PKG_DEVLIB_DIR	= $(DESTDIR)@libexecdir@@libdirsuffix@
+PKG_INC_DIR	= $(DESTDIR)@includedir@/attr
+PKG_MAN_DIR	= $(DESTDIR)@mandir@
+PKG_DOC_DIR	= $(DESTDIR)@datadir@/doc/@pkg_name@
 PKG_LOCALE_DIR	= @datadir@/locale
 
 CC		= @cc@
